modules/hosted-cluster-single-node-recovery.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assembly:
+//
+// * hcp-backup-restore-dr.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-cluster-single-node-recovery_{context}"]
+= Recovering an etcd member for a hosted cluster
+
+An etcd member of a 3-node cluster might fail because of corrupted or missing data. To recover the etcd member, complete the following steps.
+
+.Procedure
+
+. If you need to confirm that the etcd member is failing, enter the following command:
++
+[source,terminal]
+----
+$ oc get pods -l app=etcd -n $CONTROL_PLANE_NAMESPACE
+----
++
+The output looks like this example if the etcd member is failing:
```

Comment:
Is _looks_ anthropomorphic? 

> The output appears similar to the following if the etcd member is failing:

Reply:
I don't think _looks_ is anthropomorphic unless the intention is to say that the output is looking at something. I can change it to "The output resembles this example..."